pkg/kubelet/metrics/metrics.go

```diff
@@ -349,7 +349,7 @@ var (
 	RunningPodCount = metrics.NewGauge(
```

Comment:
not critical at all. This name change changes from `pod` (singular) to `pods` (plural) so maybe the same rename can be done for the variable name.

Reply:
Thanks for reminding, I never thought about the variable names, just focusing the metrics...But I guess it's optional.